### ISSUE TITLE
Typo in application howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7637]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7637|
|Type|enhancement|
|Created|2014-09-15 04:30:38Z|
|Branch|https://github.com/twisted/twisted/tree/cleanup-docs-7637|

It causes the rst to misrender

Attachments:

* [t7637.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/583/5832a25973c7a9a28b77bd5ad065fd9570df3aa0/1c4a8d8ad26a1c41d5cf538778d70f305a1eafaa.diff) (603 bytes) - added by Alex on 2014-09-15 04:31:08Z - 
* [t7637.2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/583/5832a25973c7a9a28b77bd5ad065fd9570df3aa0/a8f5fff02fc234bd93f6c4cb0656fc2e38201807.diff) (2590 bytes) - added by Alex on 2014-09-15 04:37:57Z - I missed a few violations the first time.

<details><summary>Searchable metadata</summary>

```
trac-id__7637 7637
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_cleanup_docs_7637 branches/cleanup-docs-7637
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1410755438220374 1410755438220374
changetime__1474198049187875 1474198049187875
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex set owner to @glyph|
|-|-|

There was supposed to be an empty `.misc` topfile, but trac doesn't seem to want to show it.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

Looks good. Thanks! Ready to merge!

Yep... trac diff viewers is stupid and does not show empty files in diff.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

(In [43076]) Branching to cleanup-docs-7637.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [43078]) Merge cleanup-docs-7637: Fix up Application Howto typos

Author: Alex
Reviewer: adiroiban
Fixes: [#7637](https://github.com/twisted/twisted/issues/7637)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
